ocs_ci/ocs/ui/mcg_ui.py

```diff
@@ -248,16 +248,56 @@ def delete_bucketclass_ui(self, bc_name):
         self.do_click(self.generic_locators["confirm_action"])
 
 
-class ObcUI(PageNavigator):
+class BucketsUI(PageNavigator):
+    """
+    A class representation for abstraction of OBC or OB-related OpenShift UI actions
+
+    """
+
+    def __init__(self, driver):
+        super().__init__(driver)
+        self.ocs_version = str(version.get_ocs_version_from_csv(only_major_minor=True))
+        self.obc_loc = locators[self.ocs_version]["obc"]
+
+    def select_openshift_storage_default_project(self):
+        """
+        Helper function to select openshift-storage project
+
+        Notice: the func works from PersistantVolumeClaims, VolumeSnapshots and OBC pages
+        """
+        logger.info("Select openshift-storage project")
+        self.do_click(self.generic_locators["project_selector"])
+        self.wait_for_namespace_selection(
+            project_name=config.ENV_DATA["cluster_namespace"]
```

Comment:
Do you think we should keep it flexible to select any project?

Reply:
done, project is an attribute